omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -23,6 +23,7 @@
             conf_dir = "#{conf_dir_root}/extra_package_files/EXAMPLECONFSLOCATION"
             mkdir conf_dir
             move "#{install_dir}/etc/datadog-agent/datadog.yaml.example", conf_dir_root, :force=>true
+            move "#{install_dir}/etc/datadog-agent/root.json", conf_dir_root, :force=>true
```

Comment:
I just want to note that [this](https://github.com/DataDog/integrations-core/blob/dc7e40c3083240676fc68020785e46bf44c123c3/.tuf-root.json) is *not* the _final_ TUF root metadata. We will be performing an offline root key generation ceremony later this month...

Reply:
Great, that's what I was thinking was going to happen too, glad we're on the same page :) Now, if we need smooth transition of keys, I'm going to need a copy of these keys around. If not, and we're willing to break a smooth upgrade for customers who should not yet be testing this alpha feature, then we can document manual workarounds (delete old metadata, install a copy of the new metadata, and try again).

Reply:
Hmm, I noticed that the `repositories/public-integrations-core/targets` is not being created. Maybe TUF will create this directory by default, but we shouldn't depend on it.